R/paramatch.r

```diff
@@ -14,23 +14,24 @@
 #' @examples
 #' mp <- mungebits::mungeplane(data.frame(id = c(1:2), text = c("This is an example.", "Great code uses examples like this example."), stringsAsFactors = FALSE))
 #' mb <- mungebits:::mungebit(paramatch)
-#' mb$run(mp, col = 'text', top_n = 2)
+#' mb$run(mp, col = 'text', top_n_words = 2)
```

Comment:
I switched top_n and top_n_words because I think it makes more sense for top_n = numeric and top_n_words = c(words)